docs/docs/noir/concepts/control_flow.md

```diff
@@ -46,9 +45,26 @@ Alternatively, `start..=end` can be used for a range that is inclusive on both e
 
 The index for loops is of type `u64`.
 
+## Loops
+
+In unconstrained code, `loop` is allowed for loops that end after a certain condition is met
+(or to loop forever):
+
+```rust
+let mut i = 10
+loop {
```

Comment:
Maybe this should be introduced after the `Break and Continue` section since it requires break at least

Reply:
Done! I also added the same explanation as `break` and `continue` about why `loop` can't be used in constrained code.